pytest_rts/utils/git.py

```diff
@@ -1,26 +1,40 @@
 """This module contains code for git operations"""
 import re
 import subprocess
-from typing import Dict, List, Set
+from typing import List, Set
 
 from pydriller import GitRepository
 
 
-def get_changed_files_current(repo: GitRepository) -> List[str]:
+def commit_exists(commithash: str, repo: GitRepository) -> bool:
+    """Check if a given commithash exists in the Git repository"""
+    if not commithash:
+        return False
+    return commithash in [commit.hash for commit in repo.get_list_commits()]
```

Comment:
* it is better to use `repo.get_commit(commithash)` here
* print warning if `commithash` is provided but doesn't exist in history